.github/ISSUE_TEMPLATE/3-documentation.yml

```diff
@@ -0,0 +1,30 @@
+name: "\U0001F4DA Documentation"
+description: "Suggest a change or new page to be added to re-pack.dev"
+labels: [status:new, type:documentation]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this issue!
+  - type: checkboxes
+    id: documentation_is
+    attributes:
+      label: Documentation is
+      options:
+        - label: Missing
+        - label: Outdated
+        - label: Confusing
+        - label: Not sure?
+  - type: textarea
+    id: description
+    attributes:
+      label: Explain in Detail
+      description: A clear and concise description of your suggestion.
+      placeholder: The description of ... page is not clear. I thought it meant ... but it wasn't.
+    validations:
+      required: true
+  - type: textarea
+    id: suggestion
+    attributes:
+      label: Your Suggestion for Changes
+    validations:
+      required: true
```

Comment:
This is nicely steering someone into sending a PR with a solution. Maybe we can ask them something along the lines of "Hey, maybe you'd like to contribute this change? If so please open a Pull Request"

Reply:
I like that, will include it!